google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -387,6 +406,7 @@ public void close() {
     }
 
     try {
+      log.info("Begin shutting down user callback thread pool for stream " + streamName);
```

Comment:
Is this logging accurate? with multiplexing, the identifier shouldn't be streamName. It is not just for this change but in general, seems streamName is not good enough to indicate a connection.

Also seems we should change this log to fine level.

Reply:
Yes, we need a new kind of identifier besides stream name 

let's change them together in the next PR

changed to fine